langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic/AnthropicChatModel.java

```diff
@@ -219,6 +225,16 @@ public AnthropicChatModelBuilder toolChoice(ToolChoice toolChoice) {
             return this;
         }
 
+        public AnthropicChatModelBuilder toolNameChoice(String toolNameChoice) {
```

Comment:
```suggestion
        public AnthropicChatModelBuilder toolChoiceName(String toolChoiceName) {
```
Seems to be a more appropriate name (`tool_choice.name`), WDYT?
If you are going to change it, please ensure it is changed everywhere, also in the documentation